app/modules/MenuCustom/MenuCustom.tsx

```
@@ -20,67 +20,49 @@ import {
-  const [categories, setCategories] = useState<Category[]>([
-    { id: '1', name: 'Pizza', description: 'Delicious Italian pizzas' },
-    { id: '2', name: 'Burgers', description: 'Juicy beef and chicken burgers' },
-    { id: '3', name: 'Salads', description: 'Fresh and healthy salads' },
-    { id: '4', name: 'Desserts', description: 'Sweet treats and desserts' },
-    { id: '5', name: 'Beverages', description: 'Refreshing drinks' },
-  ]);
+  const { categories, setCategories } = useCategoryStore();
   const { menuItems, setMenuItems } = useMenuItemStore();
+  const { comboMeals, setComboMeals } = useComboMealStore();
 
   const { isPending, isError, isSuccess, data, error } = useQuery({
```

ESLint

[error] 'isPending' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)